### PR TITLE
SendReconciliationReportProperties support

### PR DESCRIPTION
**Pull request by ftdholbrook**
RP-473 multi-tenant task SendReconciliationReportProperties support
- moved ArchiveProperties to multi-tenant package to avoid circular dependency

### CODE REVIEW

File: multi-tenant/src/test/resources/application-tenant_ca.yml [hunk 1]
```diff
@@ -44,3 +44,14 @@ task:
           oauth2:
             username: ca.user@example.com
             password: caPwd!
+  send-reconciliation-report:
+    tenants:
+      CA:
+        log: false
+        archives:
+          - uri-root: file:///tmp/
```
Comment (esotrope) [inline multi-tenant/src/test/resources/application-tenant_ca.yml]: woah what is this magic yaml notation?

**Comment by ftdholbrook** [comment]
move the multi-tenant archive stuff to the multi-tenant package because of a circular dependency problem.  I left the package name the same, and the archive package depends on the multi-tenant package... so it should have no real impact.

File: multi-tenant/src/test/resources/application-tenant_ca.yml (hunk 1 above)
Comment (ftdholbrook) [reply multi-tenant/src/test/resources/application-tenant_ca.yml]: list of objects?

Comment (ftdholbrook) [reply multi-tenant/src/test/resources/application-tenant_ca.yml]: curious to see how Jackson is going to handle it on writes...

Comment (esotrope) [reply multi-tenant/src/test/resources/application-tenant_ca.yml]: ah i see. the first one not having the other props threw me

**Pull request merged**